CHANGES

```diff
@@ -22,12 +26,45 @@ v2019.1-dev 2019-06-04
 	--msl-discrete-descriptor-set=<number>
 	--hlsl-enable-compat
    - Reintroduce shaderc_spvc_compile_options_set_shader_model
- - Infrastructure
+   - Added option to inject robust buffer access code
+   - Added support for emitting ToVulkan shaders
+   - Added spirv-opt based IR generation as alternative to built in spirv_cross IR gen
+   - Added API for specifying source and target execution environments
+   - Added option & reflection API methods neede by Dawn
+   - Substantial internal refactoring and code cleanup
+   - Large number of breaking changes to the API
+      -  Replaced shaderc_spvc_compile_options_set_shader_model with
+         shaderc_spvc_compile_options_set_hlsl_shader_model
+      - Compiler initialization and shader generation moved into seperate calls
+      - Seperated return codes from shaderc ones
+      - Many small API changes
+   - Improvements to testing
+      - Refactoring and clean up of run_spirv_cross_tests.py
+      - Seperation of expectations into known failures, known invalids, and cases
+        that need investigation
+      - Tweaks and fixes to substantially increase passing cases
+      - Added support for running tests using spvc IR generation
+   - Infrastructure
    - Update DEPS with cross-verified commits from associated repos.
    - Add utils/roll-deps
+ - Infrastructure:
+   - Removed Appveyor artifacts link
+   - Improvements and clean up of DEPS rolling scripts
+   - Enabled warnings about implicit fallthrough
+   - Enabled improper semicolon warnings
+   - Restricted -fPIC to platforms that support it
+   - Converted remaining scripts to use Python 3
+   - Replaced nosetest with unittest
+   - Removed assumptions aabout location of spirv-tools, effcee, and RE2
+   - Migrated BUILD.gn spirv_cross dependency to local repo
  - Fixes:
    - Fix duplicate install paths when using VisualStudio
    #666: Update Docker file to use Python3
+   - Fixed BUILD.gn for Chromium, Dawn & Fuchsia
```

Comment:
Please move these above the #666 
Otherwise, it looks like the next 6 lines of changes were part of the fix for #666 

(My rule is:    fixes that have no issue number go first; then followed by fixes with specific issue numbers.  And I don't dig too far to find issue numbers).

Reply:
Done